README.md

```diff
@@ -19,6 +19,10 @@
 Would you like to learn about Rhino hands-on? Join our events!
 Upcoming events will be added to this section - stay tuned.
 
+* [**Open Source Spotlight: New developments in Rhino**](https://events.ringcentral.com/events/shiny-gathering-11-b3775827-5b59-4068-ab59-82d9a262b7f0)<br>
+2024-02-27 at 19:00 (UTC+2)<br>
```

Comment:
```suggestion
2024-02-27 at 18:00 (UTC+1)<br>
```

The convention here is to use the local time of the event, which is 6 PM CET (UTC+1).